### ISSUE TITLE
haze.exe not showing any cmdline options

### ISSUE THREAD

**Issue by k0ss** [richinseattle/haze#2]
```
C:\haze\build\Release>haze
Haze Binary Fuzzer

Usage:

Attach:
haze <options> -pid <pid to attach to>
Run:
haze <options> -- <target command line>

Options:


C:\haze\build\Release>
```

I'll take a look through the source and see if I can figure out what's going on.

**Comment by k0ss** [comment]
Ah ok I see the options parsing is copied over from `tinyinst-coverage.cpp` and they don't show the options in the usage string either.  Guess based on the readme I thought it'd show.  Once I have a dev env I can do a quick PR if you want the usage string to be consistent with the readme.

**Comment by k0ss** [comment]
Fixed in #3
